src/components/select-field/SelectField.scss

```diff
@@ -12,6 +12,23 @@
     }
 }
 
+.search-input-container {
+    width: 100%;
+    padding-right: $bdl-grid-unit;
+    padding-bottom: 8px;
+    padding-left: $bdl-grid-unit;
+
+    input {
+        width: 100%;
+    }
+}
+
+.no-results {
+    padding-top: 5px;
```

Comment:
these are not using the grid either. can you use our variable for padding/margin values? Also consider a more specific class name (or as suggested above, use/create a disabled datalist item so you get styling for free

Reply:
Yep I can add the following:
```
.select-option.is-disabled {
    pointer-events:none;
}
```